crates/bevy_gltf/src/loader.rs

```diff
@@ -485,7 +485,9 @@ async fn load_buffers(
             gltf::buffer::Source::Uri(uri) => {
                 if uri.starts_with("data:") {
                     if uri.starts_with(OCTET_STREAM_URI) {
-                        buffer_data.push(base64::decode(&uri[OCTET_STREAM_URI.len()..])?);
+                        buffer_data.push(base64::decode(
+                            &uri.strip_prefix(OCTET_STREAM_URI).unwrap(),
```

Comment:
Isn't this unwrap exactly the check above. I.e. you should map the error into `GtlfError::BufferFormatUnsupported`.

Reply:
Do you mean this with map?
```
                    if let Some(stripped_uri) = &uri.strip_prefix(OCTET_STREAM_URI) {
                        buffer_data.push(base64::decode(stripped_uri)?);
                    } else {
                        return Err(GltfError::BufferFormatUnsupported)
                    }
```

Reply:
More
```rust
buffer_data.push(base64::decode(uri.strip_prefix(OCTET_STREAM_URI).ok_or(GltfError::BufferFormatUnsupported)?)?);
```